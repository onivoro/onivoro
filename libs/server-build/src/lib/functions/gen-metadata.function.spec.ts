import { genMetadata } from './gen-metadata.function';

describe('genMetadata', () => {
  it.each([
    ['api-account', '314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve', true],
    ['api-compliance', '314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve', true],
    ['cli-evo', '314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve', true],
    ['api-account', '314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve', false],
    ['api-compliance', '314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve', false],
    ['cli-evo', '314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve', false],
  ])('worx', (app, ecr, isProd) => {
    expect(genMetadata(app, ecr, isProd)).toMatchSnapshot();
  });
});