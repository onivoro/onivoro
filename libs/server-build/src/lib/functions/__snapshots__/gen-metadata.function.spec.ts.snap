// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`genMetadata worx 1`] = `
Object {
  "repoColonTag": "314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve:api-account",
}
`;

exports[`genMetadata worx 2`] = `
Object {
  "repoColonTag": "314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve:api-compliance",
}
`;

exports[`genMetadata worx 3`] = `
Object {
  "repoColonTag": "314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve:cli-evo",
}
`;

exports[`genMetadata worx 4`] = `
Object {
  "repoColonTag": "314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve:api-account-staging",
}
`;

exports[`genMetadata worx 5`] = `
Object {
  "repoColonTag": "314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve:api-compliance-staging",
}
`;

exports[`genMetadata worx 6`] = `
Object {
  "repoColonTag": "314218077608.dkr.ecr.us-east-2.amazonaws.com/evolve:cli-evo-staging",
}
`;
